test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -135,6 +135,24 @@ class SkipTest(Exception):
 TIMEOUT_KEY = '__testcase_timeout__'
 
 
+def skip(reason=None):
+    """
+    Unconditionally skip a test.
+
+    :param reason: Reason for the skip
+    :type reason: str or None
+    """
+    skip_flag = True
+    if reason is None:
+        reason = "Skipped test execution"
+
+    def decorator(test_item):
```

Comment:
The other decorators have a function called 'wrapper' inside them.  This one has 'decorator'.  Does it matter?  Should we be consistent?